Exception message for invalid token is unclear

```
$ cat Test.java 
public class Test {
   private Optional<String> field1;
}

$ cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
          "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
    <module name="TreeWalker">
        <module name="IllegalType">
            <property name="illegalClassNames" value="java.util.Optional" />
            <property name="tokens" value="VARIABLE_DEF" />
            <property name="memberModifiers" value=""/>
        </module>
   </module>
</module>

$ java -jar checkstyle-8.36.2-all.jar -c config.xml Test.java
com.puppycrawl.tools.checkstyle.api.CheckstyleException: cannot initialize module TreeWalker - cannot initialize module IllegalType - Cannot set property 'memberModifiers' to ''
	at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:482)
	at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:201)
	at com.puppycrawl.tools.checkstyle.Main.runCheckstyle(Main.java:404)
	at com.puppycrawl.tools.checkstyle.Main.runCli(Main.java:331)
	at com.puppycrawl.tools.checkstyle.Main.execute(Main.java:190)
	at com.puppycrawl.tools.checkstyle.Main.main(Main.java:125)
Caused by: com.puppycrawl.tools.checkstyle.api.CheckstyleException: cannot initialize module IllegalType - Cannot set property 'memberModifiers' to ''
	at com.puppycrawl.tools.checkstyle.TreeWalker.setupChild(TreeWalker.java:125)
	at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:201)
	at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:477)
	... 5 more
Caused by: com.puppycrawl.tools.checkstyle.api.CheckstyleException: Cannot set property 'memberModifiers' to ''
	at com.puppycrawl.tools.checkstyle.api.AutomaticBean.tryCopyProperty(AutomaticBean.java:241)
	at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:194)
	at com.puppycrawl.tools.checkstyle.TreeWalker.setupChild(TreeWalker.java:120)
	... 7 more
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.commons.beanutils.PropertyUtilsBean.invokeMethod(PropertyUtilsBean.java:2128)
	at org.apache.commons.beanutils.PropertyUtilsBean.setSimpleProperty(PropertyUtilsBean.java:2109)
	at org.apache.commons.beanutils.BeanUtilsBean.copyProperty(BeanUtilsBean.java:437)
	at com.puppycrawl.tools.checkstyle.api.AutomaticBean.tryCopyProperty(AutomaticBean.java:231)
	... 9 more
Caused by: java.lang.IllegalArgumentException: given name 
	at com.puppycrawl.tools.checkstyle.utils.TokenUtil.getTokenId(TokenUtil.java:174)
	at com.puppycrawl.tools.checkstyle.checks.coding.IllegalTypeCheck.setMemberModifiers(IllegalTypeCheck.java:828)
	... 17 more
Checkstyle ends with 1 errors.
```

one more case - https://github.com/checkstyle/sonar-checkstyle/pull/338#issuecomment-705581364 

Expected:
`java.lang.IllegalArgumentException: unknown TokenTypes value '' `

--- a/src/main/java/com/puppycrawl/tools/checkstyle/utils/TokenUtil.java
+++ b/src/main/java/com/puppycrawl/tools/checkstyle/utils/TokenUtil.java
@@ -48,11 +48,11 @@ public final class TokenUtil {
     /** Array of all token IDs. */
     private static final int[] TOKEN_IDS;
 
-    /** Prefix for exception when getting token by given id. */
-    private static final String TOKEN_ID_EXCEPTION_PREFIX = "given id ";
+    /** Format for exception message when getting token by given id. */
+    private static final String TOKEN_ID_EXCEPTION_FORMAT = "unknown TokenTypes id '%s'";
 
-    /** Prefix for exception when getting token by given name. */
-    private static final String TOKEN_NAME_EXCEPTION_PREFIX = "given name ";
+    /** Format for exception message when getting token by given name. */
+    private static final String TOKEN_NAME_EXCEPTION_FORMAT = "unknown TokenTypes value '%s'";
 
     // initialise the constants
     static {
@@ -152,11 +152,13 @@ public final class TokenUtil {
      */
     public static String getTokenName(int id) {
         if (id > TOKEN_VALUE_TO_NAME.length - 1) {
-            throw new IllegalArgumentException(TOKEN_ID_EXCEPTION_PREFIX + id);
+            throw new IllegalArgumentException(
+                String.format(Locale.ROOT, TOKEN_ID_EXCEPTION_FORMAT, id));
         }
         final String name = TOKEN_VALUE_TO_NAME[id];
         if (name == null) {
-            throw new IllegalArgumentException(TOKEN_ID_EXCEPTION_PREFIX + id);
+            throw new IllegalArgumentException(
+                String.format(Locale.ROOT, TOKEN_ID_EXCEPTION_FORMAT, id));
         }
         return name;
     }
@@ -171,7 +173,8 @@ public final class TokenUtil {
     public static int getTokenId(String name) {
         final Integer id = TOKEN_NAME_TO_VALUE.get(name);
         if (id == null) {
-            throw new IllegalArgumentException(TOKEN_NAME_EXCEPTION_PREFIX + name);
+            throw new IllegalArgumentException(
+                String.format(Locale.ROOT, TOKEN_NAME_EXCEPTION_FORMAT, name));
         }
         return id;
     }
@@ -185,7 +188,8 @@ public final class TokenUtil {
      */
     public static String getShortDescription(String name) {
         if (!TOKEN_NAME_TO_VALUE.containsKey(name)) {
-            throw new IllegalArgumentException(TOKEN_NAME_EXCEPTION_PREFIX + name);
+            throw new IllegalArgumentException(
+                String.format(Locale.ROOT, TOKEN_NAME_EXCEPTION_FORMAT, name));
         }
 
         final String tokenTypes =

--- a/src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilTest.java
+++ b/src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilTest.java
@@ -126,8 +126,8 @@ public class TokenUtilTest {
             fail("IllegalArgumentException is expected");
         }
         catch (IllegalArgumentException expectedException) {
-            assertEquals("given id " + nextAfterMaxId, expectedException.getMessage(),
-                    "Invalid exception message");
+            assertEquals("unknown TokenTypes id '" + nextAfterMaxId + "'",
+                expectedException.getMessage(), "Invalid exception message");
         }
     }
 
@@ -155,7 +155,8 @@ public class TokenUtilTest {
             fail("IllegalArgumentException is expected");
         }
         catch (IllegalArgumentException expected) {
-            assertEquals("given id " + id, expected.getMessage(), "Invalid exception message");
+            assertEquals("unknown TokenTypes id '" + id + "'", expected.getMessage(),
+                "Invalid exception message");
         }
     }
 
@@ -167,7 +168,8 @@ public class TokenUtilTest {
             fail("IllegalArgumentException is expected");
         }
         catch (IllegalArgumentException expected) {
-            assertEquals("given name " + id, expected.getMessage(), "Invalid exception message");
+            assertEquals("unknown TokenTypes value '" + id + "'", expected.getMessage(),
+                "Invalid exception message");
         }
     }
 
@@ -179,7 +181,8 @@ public class TokenUtilTest {
             fail("IllegalArgumentException is expected");
         }
         catch (IllegalArgumentException expected) {
-            assertEquals("given name " + id, expected.getMessage(), "Invalid exception message");
+            assertEquals("unknown TokenTypes value '" + id + "'", expected.getMessage(),
+                "Invalid exception message");
         }
     }
 

